client/js/components/hyperdrive/index.js

```diff
@@ -24,7 +24,7 @@ if (module.parent) {
       if (entry.type === 'directory') {
         send('archive:update', {root: entry.name})
       } else {
-        send('preview:file', {archiveKey: state.archive.key, entry: entry.name}, noop)
+        send('preview:file', {archiveKey: state.archive.key, fileName: entry.name}, noop)
```

Comment:
Entry is an object that has a name properly but fileName is just a string,
that's why I thought to name them differently.

El martes, 30 de agosto de 2016, Lauren Garcia <notifications@github.com>
escribió:

> In client/js/components/hyperdrive/index.js
> <https://github.com/datproject/dat.land/pull/205#discussion_r76697028>:
>
> > @@ -24,7 +24,7 @@ if (module.parent) {
> >        if (entry.type === 'directory') {
> >          send('archive:update', {root: entry.name})
> >        } else {
> > -        send('preview:file', {archiveKey: state.archive.key, entry: entry.name}, noop)
> > +        send('preview:file', {archiveKey: state.archive.key, fileName: entry.name}, noop)
>
> only comment, otherwise all looks good: maybe we should stick with
> original nomenclature around archive, so instead of state.archive.fileName,
> we go for state.archive.entry. i stubbed it out poorly in the model :)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/datproject/dat.land/pull/205/files/5a396f72f356a7f0ca23c5b88d786be2697e14e3#r76697028>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAmotImVvx0qNUokcUWqTNOBKt23mQAtks5qk1gMgaJpZM4JvUSA>
> .
>


-- 
Karissa McKelvey
http://karissa.github.io/

Dat Data
http://dat-data.com